### ISSUE TITLE
Missing libnss3

### ISSUE THREAD

**Issue by mcolyer** [heroku/heroku-buildpack-xvfb-google-chrome#7]
When I attempt to use the current master branch of this buildpack it selenium (via Capybara in ruby) fails to connect. It returns:

```
Selenium::WebDriver::Error::UnknownError: unknown error: Chrome failed to start: exited abnormally
  (Driver info: chromedriver=2.25.426924 (649f9b868f6783ec9de71c123212b908bf3b232e),platform=Linux 3.13.0-112-generic x86_64)
```

I found a [fork](https://github.com/kevinsawicki/heroku-buildpack-xvfb-google-chrome/) that includes libnss3 and that seems to fix it. Would you consider merging that into master?

**Comment by simzen85** [comment]
@mcolyer Thanks a lot, I was struggled with the official buildpack until I read your answer

**Comment by frbry** [comment]
This fixed my problems as well. Thanks @mcolyer !

This should be merged into `master`.

**Comment by joshwlewis** [comment]
Sorry for the delay, y'all. `libnss3` is included as of d757eecb1b1c0dd5b091ade0a0ad772e378e966f.
